packages/office-ui-fabric-react/src/components/Popup/Popup.tsx

```diff
@@ -46,7 +46,7 @@ export class Popup extends BaseComponent<IPopupProps, {}> {
       // apply the correct focus. Without the setTimeout, we end up focusing the correct thing, and then React wants
       // to reset the focus back to the thing it thinks should have been focused.
       setTimeout(() => {
-        if (this._originalFocusedElement) {
+        if (this._originalFocusedElement && getDocument().activeElement === getDocument().body) {
```

Comment:
I am just a bit nervous that if for some reason, the activeElement is not set to body, this change may not restore the focus back to original element?  without this change, basically we cannot have nested callout.  With this change there may be slight change restore focus could be impacted even for none nested callout path way?